src/main/java/com/ibm/cloud/sdk/core/http/HttpClientSingleton.java

```diff
@@ -263,6 +264,31 @@ protected SSLSocket configureSocket(SSLSocket socket) throws IOException {
     }
   }
 
+  /**
+   * Sets a new list of interceptors for the specified {@link OkHttpClient} instance by removing the specified
+   * interceptor and returns a new instance with the interceptors configured as requested.
+   *
+   * @param client the {@link OkHttpClient} instance to set the proxy authenticator on
+   * @param interceptorToRemove the class name of the interceptor to remove
+   * @return the new {@link OkHttpClient} instance with the new list of interceptors
+   */
+  private OkHttpClient reconfigureClientInterceptors(OkHttpClient client, String interceptorToRemove) {
+    OkHttpClient.Builder builder = client.newBuilder();
+
+    if (!builder.interceptors().isEmpty()) {
+      for (int i = 0; i < builder.interceptors().size(); i++) {
+        String currentInterceptor = builder.interceptors().get(i).getClass().getSimpleName();
+        if (currentInterceptor.equals(interceptorToRemove)) {
+          LOG.log(Level.INFO, "Removing interceptor" + currentInterceptor + " from http client instance.");
+          System.out.println("Removing interceptor" + currentInterceptor + " from http client instance.");
```

Comment:
I don't think we need to log to the logger AND stdout, do we?

Reply:
Nope, that was a remnant of my debugging. I'll remove it.